hw/xbox/mcpx_apu.c

```diff
@@ -872,9 +1095,61 @@ static void mcpx_apu_realize(PCIDevice *dev, Error **errp)
     pci_register_bar(&d->dev, 0, PCI_BASE_ADDRESS_SPACE_MEMORY, &d->mmio);
 
 
-    d->se.frame_timer = timer_new_ms(QEMU_CLOCK_VIRTUAL, se_frame, d);
+    d->se.frame_timer = timer_new_ms(QEMU_CLOCK_VIRTUAL, process_all_se_frames, d);
     d->gp.dsp = dsp_init(d, gp_scratch_rw, gp_fifo_rw);
     d->ep.dsp = dsp_init(d, ep_scratch_rw, ep_fifo_rw);
+
+
```

Comment:
Add comment that only debug functionality is found beyond this point, or something?